trident/absorption_spectrum/absorption_spectrum.py

```diff
@@ -683,18 +685,20 @@ def _add_lines_to_spectrum(self, field_data, use_peculiar_velocity,
             # quantities for the dictionary are combined correctly. 
                 comm = _get_comm(())
                 if comm.size > 1:
-                    obs_dict_fields = [column_density,tau_ray,delta_lambda,
+                    obs_dict_fields = [column_density,tau_ray,EW, delta_lambda,
                                        lambda_obs, thermal_b, thermal_width]
                     obs_dict_fields = [comm.mpi_allreduce(field,op="sum") for field in obs_dict_fields]
 
-                 # Update the line_observables_dict with values for this line
-                obs_dict = {"column_density":column_density,
+                # Update the line_observables_dict with values for this line
+                EW = np.sum(1-np.exp(-current_tau_field))*self.bin_width
+                obs_dict = {"column_density_ray":column_density,
+                            "column_density":np.sum(column_density),
                             "tau_ray":tau_ray,
```

Comment:
Specifically, this is what I'm referring to.

Reply:
I see. So would I be able to get the EW from some specific ion transition by summing over the tau_ray? I remember not being sure the math checked out on that. The sum of many equivalent widths would not be the same as the equivalent width of the sum unless all the elements of the ray are in the linear regime, the linear regime is legitimately perfectly linear, and the final profile of the whole line is also still in the linear regime.

I basically wanted to figure out the EW before putting together several different lines, but after combining all the boxes in the sightline, so I wouldn't have to assume this.

Reply:
OK, so I think a confusing issue related to all of this is that I chose `EW` as a variable name for some stuff in the subgrid deposition code which was dealing with Equivalent Widths in Tau, not equivalent with in flux.  After this PR gets merged, I will fix this in the code to not confuse other developers (including myself!) on this in the future.

What you have done here looks for the most part correct.  I'll include a couple of notes inline that you should correct before this gets merged, but overall it looks OK.